tests/unit/core/test_views.py

```diff
@@ -164,3 +164,14 @@ def test_clear_elements_with_stable(self, mock_object):
         self.viewset.request = request
         self.viewset.clear_elements(request=request, pk=MagicMock())
         self.assertEqual(len(mock_object.return_value.config), 1)
+
+
+class WebclientJavaScriptView(TestCase):
+    def setUp(self):
+        self.request = MagicMock()
+        self.view_instance = views.WebclientJavaScriptView()
+        self.view_instance.request = self.request
+
+    def test_app_configs(self):
+        response = self.view_instance.get()
```

Comment:
Hm. Ok. What about mocking the Permissions.objects.all() call only? Then the test is only about that the key permissions occurs in the WebClientJavascriptView as AngularJS constant.